scripts/darwin.sh

```diff
@@ -1,35 +1,38 @@
-#!/bin/sh
+#!/bin/bash
 
-function get_key {
-  echo $(grep "$1" | awk -F "  +" '{ print $3 }')
+set -u
+set -e
+
+get_key() {
+  grep "$1" | awk -F "  +" '{ print $3 }'
```

Comment:
You need to revert the change on this line - it's **deliberately** unquoted, see #131